crates/wasm-mutate/src/mutators/function_body_unreachable.rs

```diff
@@ -23,6 +23,10 @@ impl Mutator for FunctionBodyUnreachable {
         let mut reader = CodeSectionReader::new(code_section.data, 0)?;
 
         let count = reader.get_count();
+        if count == 0 {
+            return Err(Error::no_mutations_applicable());
+        }
```

Comment:
Instead of adding a condition here could the `can_mutate` function be updated below to test that there are actual functions in the module? I think right now `has_code` only tests for the presence of the code section, and the code section could be empty?

Reply:
Yeah I can do that instead.